.github/workflows/build.yml

```diff
@@ -187,15 +187,15 @@ jobs:
       - name: Build archive
         run: |
           # not sure why we're building this one
-          # mv aarch64-unknown-linux-gnu/libbindings_ffi.so jniLibs/arm64-v8a/
+          # mv aarch64-unknown-linux-gnu/libxmtpv3.so jniLibs/arm64-v8a/
           # mkdir -p jniLibs/x86_64
-          # mv x86_64-linux-android/libbindings_ffi.so jniLibs/x86_64/
+          # mv x86_64-linux-android/libxmtpv3.so jniLibs/x86_64/
```

Comment:
This one didn't seem to build when I tried, do we want this target eventually?

Reply:
These appear to adding confusion more than anything. I've removed them for now.